src/PresentationalComponents/SystemsTable/SystemsTable.js

```diff
@@ -275,7 +264,7 @@ const SystemsTable = () => {
             filters,
             workloads,
             SID,
-            // selectedTags,
+            selectedTags,
```

Comment:
it seems like this value is not used, and should be removed

Reply:
ahh good catch, didnt notice that